doc/nrf/links.txt

```diff
@@ -83,6 +83,10 @@
 
 .. _`Android CHIPTool`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/ff13bd/src/android/CHIPTool
 .. _`Commissioning nRF Connect Accessory using Android CHIPTool`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/ff13bd/docs/guides/nrfconnect_android_commissioning.md
+.. _`Building and programming OpenThread RCP firmware`: https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/nrfconnect_android_commissioning.md#building-rcp-firmware
```

Comment:
we agreed that we will use links to our sdk-connectedhomeip fork instead of CHIP upstream and to a specific commit to make sure that the links never get invalid. Could you align these links with the ones above and use `https://github.com/nrfconnect/sdk-connectedhomeip/blob/ff13bd` prefix?